mvc-api/mvc-service/app/managers/ReadJson.java

```diff
@@ -0,0 +1,42 @@
+package managers;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+public class ReadJson {
+    GetContentDefinition getContentDefinition = new GetContentDefinition();
+    public void read(String json) {
+
+        JSONParser parser = new JSONParser();
```

Comment:
We always use ObjectMapper from to parse JSON string. Please don't add new libraries.

Reply:
okay will make this change and remove the added library